packages/ketcher-core/src/domain/entities/sGroupAttachmentPoint.ts

```diff
@@ -0,0 +1,29 @@
+import assert from 'assert'
+
+export class SGroupAttachmentPoint {
```

Comment:
I propose to add comments for this class. I guess, it can be unobvious what `leaveAtomId` and `additionalData`